README.md

```diff
@@ -1,2 +1,229 @@
+
+**Math Magicians** is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+- Make simple calculations.
+- Read a random math-related quote.
+
+---
+
+#### Learning objectives
+
+- Use React components.
+- Use React props.
+- Use React state.
+
+---
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Markup</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Dynamic</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Styles</summary>
+  <ul>
+    <li><a href="https://sass-lang.com/">SASS</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Frameworks</summary>
+  <ul>
+    <li><a href="https://react.dev/">React.js</a></li>
+  </ul>
+</details>
+
+---
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **FEATURE 1**
+- **FEATURE 2**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://iturres.github.io/Task-Manager/dist/) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```bash
+  cd my-folder
+  git clone git@github.com:ITurres/math-magicians.git
+```
+
+### Install
+
+Install this project's dependencies with:
+
+- npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+```bash
+ npm run start
+```
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### Run tests
+
+```bash
+  npm test
+```
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@ITurres](https://github.com/ITurres)
+- Twitter: [@Arthur_ITurres](https://twitter.com/ArthurIturres)
+- LinkedIn: [Arthur Emanuel G. Iturres](https://www.linkedin.com/in/arturoemanuelguerraiturres/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+---
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **future features 1**
```

Comment:
- [ ] It would be better if we remove these placeholder text and replace them with actual realistic features. This will make our README more professional